smartsim/_core/_install/builder.py

```diff
@@ -351,21 +351,6 @@ def build_onnx(self) -> bool:
     def fetch_onnx(self) -> bool:
         return self.build_onnx
 
-    def copy_tf_cmake(self) -> None:
-        """Copy the FindTensorFlow.cmake file to the build directory
-        as the version included in RedisAI is out of date for us.
-        Note: opt/cmake/modules removed in RedisAI v1.2.5
```

Comment:
If we are no longer using it (and presumably haven't since RAI 1.2.3) can we check `smartsim/_core/bin/modules/FindTensorFlow.cmake` out of our source control?